config/betterquesting/DefaultQuests/Quests/Tier1LV-AAAAAAAAAAAAAAAAAAAAAw==/OilsandorHowYouG-AAAAAAAAAAAAAAAAAAAG_w==.json

```diff
@@ -45,7 +45,7 @@
       "requiredItems:9": {
         "0:10": {
           "id:8": "gregtech:gt.blockores",
-          "Count:3": 256,
+          "Count:3": 112,
```

Comment:
why 112 ?

Reply:
3x32+16

Reply:
no make it 128 or let it 256.
You get tons of it because oil sand vein has only this ore so 112 is some wired nr.